### ISSUE TITLE
Status Bar Entry Which Shows How Many Tokens The Current Note Is.

### ISSUE THREAD

**Issue by vari0nce** [SystemSculpt/obsidian-systemsculpt-ai#20]
## Describe the enhancement
It would be helpful to know how much of your notes would fit untruncated within the context window.

Token amount for a text block (per header for example) would also be handy.

Or the Token Estimator could show the total amount of tokens for chosen context notes.

At least having tried only with local models I can't see such a thing, not sure if it'd be visible with billed APIs.

**Comment by SystemSculpt** [comment]
Yes, I'm looking into expanding the ease of use regarding understanding token limitations, these are all great suggestions and I'll be sure to implement them all. I'll keep this open until they're implemented, thanks!
